cscs-checks/libraries/magma/src/patch.txt

```diff
@@ -73,7 +73,7 @@ diff -Naur src/testing_zgemm.cpp src_patched/testing_zgemm.cpp
      printf("%%========================================================================================================\n");
 -    for( int itest = 0; itest < opts.ntest; ++itest ) {
 +//ajajajaj    for( int itest = 0; itest < opts.ntest; ++itest ) {
-+    for( int itest = 0; itest < opts.ntest; itest+=100 ) {
++    for( int itest = 0; itest < 3; itest++ ) {
```

Comment:
It seems that what you are trying to do here, you could accomplish by a command line option: `opts.ntest` seems like an option.